projects/netgrif-components-core/src/lib/view/task-view/abstract-task-view.ts

```diff
@@ -13,8 +14,9 @@ export abstract class AbstractTaskViewComponent extends AbstractViewWithHeadersC
     public tasks$: Observable<Array<TaskPanelData>>;
     public loading$: Observable<boolean>;
 
-    protected constructor(protected taskViewService: TaskViewService) {
-        super(taskViewService);
+    protected constructor(protected taskViewService: TaskViewService,
+                          protected _activatedRoute?: ActivatedRoute) {
```

Comment:
if the `super` class has an activatedRoute attribute, the `protected` keyword should not be used here as it creates a new attribute